### ISSUE TITLE
Xenobio golems getting labeled free golems and suffering from the penalties.

### ISSUE THREAD

**Issue by GlueSniffer** [tgstation/tgstation#67620]
Reporting client version: 514.1583

## Round ID:
[184527](https://scrubby.melonmesa.com/round/184527)

## Reproduction (step by step):

1. Become a luminescent slimeperson.

2. Use the greater adamantine core activation to turn yourself into an adamantine golem.

3. Create another golem shell using a bar of adamantine and some material (cardboard in this instance) and transfer your golem mind into it. (Not 100% sure if this step is necessary but I didn't notice anything until after this point)

4. Walk out of the room (area) you are currently in and watch the big yellow exclamation point show up at the top right of your screen while your everything slows to a crawl.

Bonus step 5, discover that every single golem you create as a cardboard also suffers from the free golem penalty.

Very sad, cardboard golem army's day was ruined.

**Comment by Sealed101** [comment]
runtime detected:

```
[2022-06-08 01:59:04.216] runtime error: Cannot execute null.enslave mind to creator().
 - proc name: special (/obj/effect/mob_spawn/ghost_role/human/golem/special)
 -   source file: golem_roles.dm,57
 -   usr: Christopher Robin (/mob/living/carbon/human)
 -   src: the inert servant golem shell ... (/obj/effect/mob_spawn/ghost_role/human/golem/servant)
 -   usr.loc: the floor (68,161,2) (/turf/open/floor/iron)
 -   src.loc: the floor (68,162,2) (/turf/open/floor/iron)
 -   call stack:
 - the inert servant golem shell ... (/obj/effect/mob_spawn/ghost_role/human/golem/servant): special(Christopher Robin (/mob/living/carbon/human), null)
 - the inert servant golem shell ... (/obj/effect/mob_spawn/ghost_role/human/golem/servant): create(null, "Christopher Robin")
 - the inert servant golem shell ... (/obj/effect/mob_spawn/ghost_role/human/golem/servant): create(null, "Christopher Robin")
 - the inert servant golem shell ... (/obj/effect/mob_spawn/ghost_role/human/golem/servant): attack hand(Christopher Robin (/mob/living/carbon/human), /list (/list))
 - Christopher Robin (/mob/living/carbon/human): UnarmedAttack(the inert servant golem shell ... (/obj/effect/mob_spawn/ghost_role/human/golem/servant), 1, /list (/list))
 - Christopher Robin (/mob/living/carbon/human): ClickOn(the inert servant golem shell ... (/obj/effect/mob_spawn/ghost_role/human/golem/servant), "icon-x=18;icon-y=17;left=1;scr...")
 - the inert servant golem shell ... (/obj/effect/mob_spawn/ghost_role/human/golem/servant): Click(the floor (68,162,2) (/turf/open/floor/iron), "mapwindow.map", "icon-x=18;icon-y=17;left=1;scr...")
 - 
[2022-06-08 01:59:12.285] runtime error: Cannot execute null.set assigned role().
 - proc name: special (/obj/effect/mob_spawn/ghost_role/human/golem/special)
 -   source file: golem_roles.dm,69
 -   usr: Christopher Robin (/mob/living/carbon/human)
 -   src: the inert servant golem shell ... (/obj/effect/mob_spawn/ghost_role/human/golem/servant)
 -   usr.loc: the floor (68,161,2) (/turf/open/floor/iron)
 -   src.loc: the floor (68,162,2) (/turf/open/floor/iron)
 -   call stack:
 - the inert servant golem shell ... (/obj/effect/mob_spawn/ghost_role/human/golem/servant): special(Christopher Robin (/mob/living/carbon/human), null)
 - the inert servant golem shell ... (/obj/effect/mob_spawn/ghost_role/human/golem/servant): create(null, "Christopher Robin")
 - the inert servant golem shell ... (/obj/effect/mob_spawn/ghost_role/human/golem/servant): create(null, "Christopher Robin")
 - the inert servant golem shell ... (/obj/effect/mob_spawn/ghost_role/human/golem/servant): attack hand(Christopher Robin (/mob/living/carbon/human), /list (/list))
 - Christopher Robin (/mob/living/carbon/human): UnarmedAttack(the inert servant golem shell ... (/obj/effect/mob_spawn/ghost_role/human/golem/servant), 1, /list (/list))
 - Christopher Robin (/mob/living/carbon/human): ClickOn(the inert servant golem shell ... (/obj/effect/mob_spawn/ghost_role/human/golem/servant), "icon-x=18;icon-y=16;left=1;scr...")
 - the inert servant golem shell ... (/obj/effect/mob_spawn/ghost_role/human/golem/servant): Click(the floor (68,162,2) (/turf/open/floor/iron), "mapwindow.map", "icon-x=18;icon-y=16;left=1;scr...")
```

**Comment by Sealed101** [comment]
which is the same as #73368
and fixed in #73373
